src/librustdoc/test.rs

```diff
@@ -74,7 +74,16 @@ pub fn run(options: Options) -> i32 {
         lint_caps: Default::default(),
     };
 
-    let mut test_args = options.test_args.clone();
+    let mut test_args = options.test_args
+        .iter()
+        .filter_map(|arg| {
+            if arg != "rustdoc" {
+                Some(arg.clone())
```

Comment:
Hm, what is the intent of this change? Why can't `rustdoc` be passed to libtest here?

Reply:
It was a request to specifically not pass the `rustdoc` features when running tests from a previous review. Or did I miss something?

Reply:
Uh, sure, but this seems to have nothing to do with `--cfg` unless I'm missing something? I would expect the test_args vector to be the arguments like `--nocapture` and such?

Regardless, I would want this to be done not here but where we create test_args.

Reply:
Fine by me!